Content.Shared/Tools/Components/ToolComponent.cs

```diff
@@ -18,5 +18,12 @@ public sealed class ToolComponent : Component
 
         [DataField("useSound")]
         public SoundSpecifier? UseSound { get; set; }
+
+        /// <summary>
+        /// Whether this tool will overcome checks that normally
+        /// reject tool use if the target is powered. E.g., airlock prying
+        /// </summary>
+        [DataField("forcePowered")]
+        public bool ForcePowered = false;
```

Comment:
i feel like this should be a separate comp instead of just hardcoded into tool component